client/engine/chainservice/simulated_backend_chainservice_test.go

```diff
@@ -70,13 +55,22 @@ func TestDepositSimulatedBackendChainService(t *testing.T) {
 	}
 
 	// Check that the recieved events matches the expected event
-	for i := 0; i < 2; i++ {
+	for i := 0; i < 1; i++ {
```

Comment:
Do you know why only one event is received now?

Reply:
Also, let's just focus on native token (as opposed to erc20) support. Maybe that can reduce the scope of this work?

Reply:
The original deposit had multiple assets, so I think we expected multiple events because of that. It looks like we create an `expectedEvent` for each asset: `expectedEvent := NewDepositedEvent(channelID, 2, dEvent.AssetAddress, testDeposit[dEvent.AssetAddress], testDeposit[dEvent.AssetAddress])`

Now the deposit is just an eth deposit so there is only the one event.